games-util/lutris/lutris-9999.ebuild

```diff
@@ -27,9 +27,12 @@ RDEPEND="
 	dev-python/pygobject:3[${PYTHON_USEDEP}]
 	dev-python/python-evdev[${PYTHON_USEDEP}]
 	dev-python/pyyaml[${PYTHON_USEDEP}]
+	gnome-base/gnome-desktop
 	net-libs/libsoup
+	net-libs/webkit-gtk
+	x11-apps/xgamma
 	x11-apps/xrandr
-	x11-apps/xgamma"
+"
```

Comment:
I added additional deps specified by upstream in my version: app-arch/cabextract, edia-sound/fluid-soundfont, sys-process/psmisc. I wonder if some deps are needed at build time.

Reply:
What about `media-sound/fluid-soundfont`though?